console/src/App/App.js

```diff
@@ -125,11 +121,11 @@ const AppBase = kind({
 				state.userSettings.arrangements.arrangeable = !state.userSettings.arrangements.arrangeable;
 			});
 		},
-		// onToggleProfileEdit: (ev, {updateAppState}) => {
-		// 	updateAppState((state) => {
-		// 		state.appState.showProfileEdit = !state.appState.showProfileEdit;
-		// 	});
-		// },
+		onToggleProfileEdit: (ev, {updateAppState}) => {
+			updateAppState((state) => {
+				state.appState.showProfileEdit = !state.appState.showProfileEdit;
+			});
+		},
```

Comment:
You should leave this commented for the time being, otherwise you get this console warning:
![image](https://user-images.githubusercontent.com/63341832/212493764-365de3f4-2fc1-4b0e-92f7-0bd63f13d394.png)